javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientBuilderTrait.java

```diff
@@ -105,7 +105,7 @@ private static ClientBuilderTrait createHttpTrait() {
         ServiceClientProperty pipelineProperty = new ServiceClientProperty("The HTTP pipeline to send requests " +
                 "through.", ClassType.HttpPipeline, "pipeline", false,
                 JavaSettings.getInstance().isAzureOrFluent()
-                        ? "new HttpPipelineBuilder().policies(new UserAgentPolicy(), new RetryPolicy(), new CookiePolicy()).build()"
+                        ? "new HttpPipelineBuilder().policies(new UserAgentPolicy(), new RetryPolicy()).build()"
```

Comment:
This change will not have real effect, and the dummy pipeline will not work anyway.